pcstac/pcstac/client.py

```diff
@@ -39,6 +39,18 @@ class PCClient(CoreCrudClient):
 
     extra_conformance_classes: List[str] = attr.ib(factory=list)
 
+    def absolute_asset_links(self, item: Item, base_url: str) -> Item:
+        """Convert relative asset links to absolute links"""
+        # Supporting the fields extension means skipping if assets aren't here
+        if "assets" in item and "image" in item["assets"]:
```

Comment:
Rather than relative asset href links, I think the use case we're trying to support is for `link` hrefs to be relative and replaced with the current `base_url`. For example, we're expecting future collections that have a `derivedfrom` rel type, which is a link to another item in a different collection, accessed via the STAC API. The assets in all cases should always be absolute. For `links`, I think it's safe to assume any relative path should be made absolute, and we shouldn't need to check for specirfic `rel` types

Reply:
Updating this with the proposed tests, but it appears to me that stac-fastapi already does what is desired?(here: https://github.com/stac-utils/stac-fastapi/blob/c948dbfcd12fff0939d0bc3ecb907878ee217416/stac_fastapi/pgstac/stac_fastapi/pgstac/models/links.py#L105)